.circleci/config.yml

```diff
@@ -49,6 +50,13 @@ jobs:
           name: Danger
           command: DANGER_GITHUB_API_TOKEN="15e52de81a772b174cc5""e1813d0083564c69c325" yarn danger ci
 
+  test:
+    executor: hokusai/deploy
+    steps:
+      - hokusai/setup-docker
+      - run: hokusai registry pull --tag "$CIRCLE_SHA1"
+      - run: hokusai test
```

Comment:
Use `hokusai test --no-build` here to skip the build step and use the pre-built pulled image